client/types/account-overview.d.ts

```diff
@@ -57,12 +57,3 @@ declare namespace AccountOverview {
 		isLoading: boolean;
 	}
 }
-
```

Comment:
I'm guessing this removal is because now `@woocommerce/components` has their own type declaration unlike [2 years ago when this code was added](https://github.com/Automattic/woocommerce-payments/blame/develop/client/components/deposits-information/declarations.d.ts#L61).

Reply:
Note: I haven't removed this, I've moved it from `client/deposits-information/declarations.d.ts` to a shared location: `client/deposits/declarations.d.ts`.

See my [comment](https://github.com/Automattic/woocommerce-payments/pull/5773#discussion_r1146951804) below about the library version. TLDR: we're using the [same version of `@woocommerce/components` from 2 years ago](https://github.com/Automattic/woocommerce-payments/pull/1077) 😝

Reply:
If not dangerous, we could update as part of our project since we like new stuff and we are doing extensive testing of our changes.

Reply:
> If not dangerous
There are a lot of considerations to be aware of when updating component libraries, although I'm not fully across the discussion (paJDYF-2LK-p2 paJDYF-4sH-p2).

Reply:
A good thing to get across :) 

If you get to the bottom of it, please let us all know (Helix hang or P2 or slack!) – IMO we should all have a working understanding of why we pin or how we keep up to date.